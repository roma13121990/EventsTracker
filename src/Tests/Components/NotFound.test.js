import React from 'react'
import renderer from 'react-test-renderer'
import NotFound from "../../Components/NotFound";

describe('<NotFound />', () => {

    it('Should generate snapshot for Header component', () => {
        const tree = renderer.create(<NotFound text="Not Found"/>).toJSON();
        expect(tree).toMatchSnapshot()
    })


});