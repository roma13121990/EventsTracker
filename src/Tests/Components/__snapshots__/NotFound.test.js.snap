// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> Should generate snapshot for Header component 1`] = `
<div
  className="NotFound"
>
  <p>
    Not Found
  </p>
</div>
`;
